app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -354,6 +357,46 @@ class StateFragmentPresenter @Inject constructor(
     itemList.add(contentViewModel)
     stateAdapter.notifyDataSetChanged()
   }
+  private fun checkAndAddInteraction() {
+    if (currentEphemeralState.stateTypeCase.number == EphemeralState.PENDING_STATE_FIELD_NUMBER) {
```

Comment:
Please check the enum directly:

```kotlin
if (currentEphemeralState.stateTypeCase == EphemeralState.TypeCase.PENDING_STATE)
```

Reply:
It is giving error to TypeCase as unresolved reference.